webapp/components/admin_console/admin_sidebar.jsx

```diff
@@ -10,14 +10,21 @@ import * as Utils from 'utils/utils.jsx';
 
 import AdminSidebarHeader from './admin_sidebar_header.jsx';
 import AdminSidebarTeam from './admin_sidebar_team.jsx';
-import {FormattedMessage} from 'react-intl';
+import {intlShape, injectIntl, defineMessages, FormattedMessage} from 'react-intl';
 import {browserHistory} from 'react-router/es6';
 import {OverlayTrigger, Tooltip} from 'react-bootstrap';
 import SelectTeamModal from './select_team_modal.jsx';
 import AdminSidebarCategory from './admin_sidebar_category.jsx';
 import AdminSidebarSection from './admin_sidebar_section.jsx';
 
-export default class AdminSidebar extends React.Component {
+const holder = defineMessages({
```

Comment:
I think you should consider using `Utils.localizeMessage` instead of adding the intlShape and injectIntl here.

Thanks